libs/clients/regulations/src/lib/regulations.types.ts

```diff
@@ -0,0 +1,240 @@
+declare const _RegName__Brand: unique symbol
```

Comment:
Might be good idea to make a new lib such (api-domains-regulations-types) to place this and the types in `apps/web/components/Regulations/Regulations.types.ts` since they have alot of the same types.

Reply:
ah, I'd been wondering what was the idiomatic way of sharing those types.